frontend/src/components/explorer/mapViewer/SidebarDataPanels.vue

```diff
@@ -143,12 +143,12 @@ export default {
       errorMessage: '',
       selectedElementDataKeys: {
         metabolite: [
-          { name: 'name' },
+          { name: 'id' },
           { name: 'formula' },
           { name: 'compartment' },
         ],
         gene: [
-          { name: 'name', display: 'Gene&nbsp;name' },
+          { name: 'id', display: 'Gene&nbsp;id' },
```

Comment:
I think the swapping of the `id` and `name` here looks great. 
However, maybe it is better to change `Gene id` to `Id` so that the format is in consistence with metabolite. See line 146 above.

Reply:
I agree this would be better. @mihai-sysbio was there a reason why it was 'Gene name:' vs 'Name:' for metabolite before?

Reply:
I can't recall any rationale behind this. It might because how this field is called on Ensembl ("Name"), but on Uniprot it's just "Gene". The way I see it, short and clear is good.

Reply:
Reverting back to displaying gene id in the 3d-viewer as discussed. Addressed in https://github.com/MetabolicAtlas/MetabolicAtlas/pull/565/commits/67a5fe518b9fe553b1538b145061e67876583b8b.